styles/button.css

```diff
@@ -61,3 +61,7 @@
   outline: 2px solid var(--bs-btn-focus-box-shadow); /* Outer color */
   box-shadow: 0 0 0 4px var(--stanford-black); /* Inner color */
 }
+
+.btn-group .dropdown-item.active {
```

Comment:
I think this should be set at `.dropdown-menu` I don't think we want to scope it so tightly.  The `.active` is unnecessary as the variable is only for active states.